tests/jenkins/install_tqperf.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/bash
+
+set -e
+
+# Will return != 0 if not the same so will error this script
+diff -w out.dat spk000.dat
```

Comment:
We already check the spikes in https://github.com/BlueBrain/CoreNeuron/blob/e96e6e8132c5b9d0182e306769f9cb557a7a5d11/tests/jenkins/run_corenrn.sh#L31, so I think that this script can be removed

Reply:
ok